limq-marker-server/src/main/resources/application.yml

```diff
@@ -17,6 +16,7 @@ spring:
             mode: always
             continue-on-error: 'true'
     flyway:
+        enabled: false
```

Comment:
I think we will need flyway to be enabled in order for migrations to be applied when not in the local environment

Reply:
Yes indeed, but since it wasn't set, I thought we didn't want it for now. I can set it to true.